### PR TITLE
cld 409 changes

### PR DESCRIPTION
**Pull request by barkhachoithani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -368,6 +368,8 @@ As in the previous single-node example, each node of the cluster can be accessed
 
 In the previous examples, Docker secrets files were used to specify admin credentials for the MarkLogic Server. If your environment prevents the use of Docker secrets, you can use environmental variables. This approach is less secure, but it is commonly used in development environments. This is not recommended for production environments. In order to use these environment variables in the Docker compose files, remove the secrets section at the end of the Docker compose yml file, and remove the secrets section in each node. Then replace the MARKLOGIC_ADMIN_USERNAME_FILE/MARKLOGIC_ADMIN_PASSWORD_FILE variables with MARKLOGIC_ADMIN_USERNAME/MARKLOGIC_ADMIN_PASSWORD and provide the appropriate values.
 
+Using Docker secrets, username and password information are secured when transmitting the sensitive data from Docker host to Docker containers. To prevent any attacks, the login information is not  available as an environment variable. However, these values are stored in a text file and persisted in an in-memory file system inside the container. We recommend that that you delete the Docker secrets information once the cluster is up and running.
```
Comment (twosdai) [inline README.md:371]: > We recommend that that you delete

remove the duplicate that.

**Review by twosdai — CHANGES_REQUESTED** [review]
One small comment for grammar.

**Comment by barkhachoithani** [comment]
> FYI:
> 
> In our GIT flow meeting we wanted to not allow merges from dev forks directly into develop, for this PR its not a big deal, but we should create a branch in the shared repository with the same name `feature/cld-409` and merge from a fork into that in the future.
> 
> For cld-391 that was the process I followed.

I will make sure to follow the git flow nect time.

File: README.md (hunk 1 above)
Comment (barkhachoithani) [reply README.md]: good catch, removed duplicates.

**Pull request merged**